garage/envs/wrappers/atari_env.py

```diff
@@ -0,0 +1,27 @@
+"""Episodic life wrapper for gym.Env."""
+import gym
+import numpy as np
+
+
+class AtariEnv(gym.Wrapper):
+    """
+    Atari environment wrapper for gym.Env.
+
+    This wrapper convert the observations returned from baselines wrapped
+        environment, which is a LazyFrames object into numpy arrays.
+
+    Args:
+        env (gym.Env): The environment to be wrapped.
+    """
+
+    def __init__(self, env):
+        super().__init__(env)
+
+    def step(self, action):
+        """gym.Env step function."""
+        obs, reward, done, info = self.env.step(action)
+        return np.asarray(obs), reward, done, info
```

Comment:
is `obs` not an np.ndarray?

`np.asarray()` copies (AFAIK), so you will want to avoid the copy operation if you can.

Reply:
obs is not a `np.ndarray`. It is stated in the docstring that `This wrapper convert the observations returned from baselines wrapped environment, which is a LazyFrames object into numpy arrays.`